src/irmin-pack/import.ml

```diff
@@ -24,9 +24,20 @@ module Int63 = struct
   include Optint.Int63
 
   let t = Irmin.Type.int63
+
+  module Syntax = struct
+    open struct
+      module Int63 = Optint.Int63
+    end
+
+    let ( + ) = Int63.add
+    let ( - ) = Int63.sub
+    let ( < ) a b = Int63.compare a b < 0
+    let ( <= ) a b = Int63.compare a b <= 0
+    let ( > ) a b = Int63.compare a b > 0
+    let ( >= ) a b = Int63.compare a b >= 0
+    let ( === ) = Int63.equal
```

Comment:
An alternative would have been to shadow the polymorphic `( = )` from `Stdlib` but I feel that `( === )` is OK for now.

My favorite solution would be to globally shadow the polymorphic equal and replace it with `Int.equal` so that we stop using it for everything without thinking about safety and correctness. If we ever do that we can come back here and replace that triple equal to a single one too.

Reply:
On second look, the `===` didn't look great to my eyes (was just a copy-paste of what existed previously) so I changed it to just `=`. LMK if you want to revert back to using the triple equals.

I know that `Int63.equal` will work, but under what circumstances would `Stdlib.(=)` be incorrect? (I actually found one [place while doing this PR that was using `=` instead of the `===`](https://github.com/mirage/irmin/pull/1930/files#diff-a4063ff0edc6a5515331833cc6e549e87e9aae2ef5bcb8772f49a2cab07890fdL33)).

Reply:
It works for me!